src/Microsoft.SqlTools.ServiceLayer/Scripting/ScripterCore.cs

```diff
@@ -477,7 +477,21 @@ internal ScriptingScriptOperation InitScriptOperation(string objectName, string
 			    TargetDatabaseEngineEdition = GetTargetDatabaseEngineEdition(),
 			    TargetDatabaseEngineType = GetTargetDatabaseEngineType(),
 			    ScriptCompatibilityOption = GetScriptCompatibilityOption(),
-                IncludeIfNotExists = true
+                ScriptExtendedProperties = false,
+                ScriptUseDatabase = false,
+                IncludeIfNotExists = false,
```

Comment:
will these changes also apply to the Script As Create command?  Ideally Script As Create would generate the verbose CREATE syntax and Goto Definition would generate the concise CREATE statement.

Reply:
This is just for peak definition, the options are intact for script as create